Networking/Networking and working with a RESTful API.md

```diff
@@ -6,14 +6,12 @@
 If your app is going to communicate with a remote server, you will find the following list of [pods][7] useful:
 
 * [Alamofire][1]—used for HTTP networking. On the off chance that you're actually starting an Objective-C based project, you'll be working with [AFNetworking][2] instead. Both of these libraries are developed by the same team who is responsible for [nshipster.com][3]—a site you should really keep in your bookmarks.
-* [Unbox][4]—a [JSON][5] parser. In almost all cases, the data you receive from the remote server will be in a JSON format. Again, in the unlikely event of starting an Objective-C based project, you'll probably use [JSONModel][6] or [Mantle][7].
-* [UnboxedAlamofire][8]—a utility extensions for Alamofire + Unbox.
-* [Wrap][9]—a JSON encoding library. Useful for transforming models into JSON objects.
+* [CodableAlamofire][4]—an extension for Alamofire that can convert [JSON][5] into Decodable objects. In almost all cases, the data you receive from the remote server will be in a JSON format. Again, in the unlikely event of starting an Objective-C based project, you'll probably use [JSONModel][6] or [Mantle][7].
```

Comment:
Ja bi dodao da postoji i `Codable` sam po sebi koji nije 3rd party library, međutim, ako je s LE-ima dogovoreno ovako, može :+1:

Reply:
Mozda da stavim samo na kraj "you'll probably use [JSONModel][6] or [Mantle][7], otherwise use Codable".

Reply:
Ovisi o kontekstu, jer ces navedeno koristiti za Objective-C, dok ces Codable ili CodableAlamofire koristiti za Swift.

Ukratko, ovo je okej, ali bi prije ovog dijela, "Again ..." spomenuo da je moguće koristiti i Codable nativno koji je supportan od Apple-a a nije 3rd party library.

Reply:
Pitao sam LE-ove i rekli su da cak i ne treba posebno naglasavati to, ali mozemo nesto u ovom stilu? "an extension for Alamofire that can convert Encodable objects into [JSON][5] or convert JSON into Decodable objects by making use of the native swift protocols."

Reply:
Okej, štima ;)

Reply:
To znaci da bi stavio ipak ovaj text? "an extension for Alamofire that can convert Encodable objects into [JSON][5] or convert JSON into Decodable objects by making use of the native swift protocols."

Reply:
Čak niti ne, ako LE-i misle da ne treba naglašavati. Onda je prema tome u redu ovako kako si napisao u commitu.